src/Observable.ts

```diff
@@ -187,10 +187,12 @@ export default class Observable<T> implements CoreOperators<T>  {
   map: <T, R>(project: (x: T, ix?: number) => R, thisArg?: any) => Observable<R>;
   mapTo: <R>(value: R) => Observable<R>;
   materialize: () => Observable<any>;
+  max: () => Observable<any>;
   merge: (...observables: any[]) => Observable<any>;
   mergeAll: (concurrent?: any) => Observable<any>;
   mergeMap: <R>(project: ((x: T, ix: number) => Observable<any>), projectResult?: (x: T, y: any, ix: number, iy: number) => R, concurrent?: number) => Observable<R>;
   mergeMapTo: <R>(observable: Observable<any>, projectResult?: (x: T, y: any, ix: number, iy: number) => R, concurrent?: number) => Observable<R>;
+  min: () => Observable<any>;
```

Comment:
is it expected signature does not match with actual implementation?

`min<T, R>(predicate?: (x: R, y: T) => R, thisArg?: R): Observable<R>`